components/user-mgt/org.wso2.carbon.user.mgt/src/main/java/org/wso2/carbon/user/mgt/bulkimport/CSVUserBulkImport.java

```diff
@@ -133,6 +135,17 @@ public void addUserList(UserStoreManager userStore) throws UserAdminException {
         } catch (Exception e) {
             log.error(e.getMessage(), e);
```

Comment:
This is wrong. e.getMessage is anyway inside e. So you should be more explicit while giving error message.